pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,499 @@
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+    echo "${OPTION}=inserts" >> ${OUTPUT_FILE}
+    echo "${OPTION}=deletes" >> ${OUTPUT_FILE}
+    echo "${OPTION}=changes" >> ${OUTPUT_FILE}
+    echo "${OPTION}=purges" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-default-row-format" ]; then
+    echoit "  > Adding possible values dynamic, compact, redundant for option '${OPTION}' to the final list..."
+    echo "${OPTION}=dynamic" >> ${OUTPUT_FILE}
+    echo "${OPTION}=compact" >> ${OUTPUT_FILE}
+    echo "${OPTION}=redundant" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--internal-tmp-disk-storage-engine" ]; then
+    echoit "  > Adding possible values INNODB for option '${OPTION}' to the final list..."
+    echo "${OPTION}=INNODB" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-output" ]; then
+    echoit "  > Adding possible values FILE, TABLE, NONE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=NONE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-timestamps" ]; then
+    echoit "  > Adding possible values SYSTEM, UTC for option '${OPTION}' to the final list..."
+    echo "${OPTION}=UTC" >> ${OUTPUT_FILE}
+    echo "${OPTION}=SYSTEM" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--master-info-repository" ]; then
+    echoit "  > Adding possible values FILE, TABLE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--relay-log-info-repository" ]; then
+    echoit "  > Adding possible values FILE, TABLE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+    elif [ "${OPTION}" == "--windowing-use-high-precision" ]; then
+    echoit "  > Adding possible values 0, 1 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--default-storage-engine" ]; then
+    echoit "  > Adding possible value InnoDB for option '${OPTION}' to the final list..."
+    echo "${OPTION}=InnoDB" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--" ]; then
```

Comment:
Please remove all occurrences of this check

Reply:
Done